### ISSUE TITLE
First Changes: Object Plates

### ISSUE THREAD

**Issue by vhellstein** [DenverArtMuseum/biophilia-pub#28]

**Comment by vhellstein** [comment]
@1000camels these are what are currently called "Entries"

- [ ] move Kit’s name to end of essay @vhellstein 

- [ ] make these images zoomable without going into full screen mode first @mpopke 

- [ ] no caption over images  

- [ ] “Notes” header between essay and notes. @1000camels, can I do this in the Markdown files?

- [ ] Natural Analogs: Entry 2: This is the only object plate that has 2 separate objects—need to stack object info on the left + link to online collection: Create new object yaml that has these two objects in it @mpopke  

- [ ] Natural Analogs: Entry 3: Photo on left will be install photo, 3 details embedded in text with only 1 caption across: In general, only 1 photo on left, details embedded in text? @vhellstein 

- [ ] fix the wrapping of the Object Plate titles @1000camels:  h1 * selector is overly broad and applying “display: block” for some reason. Can probably just delete this rule from the CSS. 

- [ ] Object info formatted more like our tombstones: no “artist” “medium” header. @1000camels    


![Image](https://github.com/DenverArtMuseum/biophilia-pub/assets/119438504/b3ca9452-f358-460c-9414-17d2f405fb62)

**Comment by 1000camels** [comment]
I would like to sort out the urls for these. RIght now we have /natural-analogs/object-plates/Natural_Analogs_entry_A/
Instead of Natural_Analogs_entry_A, we could rename it entry-a or entry-1 or name it after the artist, sandra-davolio
@vhellstein what do you think?

**Comment by vhellstein** [comment]
@1000camels I like using the artist name for the URL so it matches the title of the page. Thanks! 

Also, I'm not feeling great these last couple of days. @mpopke, if you could jump in as needed today and tomorrow, I'd be most appreciative! I hope to be back at it tomorrow, though.

**Comment by 1000camels** [comment]
@vhellstein hope you feel better

**Comment by 1000camels** [comment]
The titles have been changed to just the artist's name, so I cannot reproduce the tombstone as you want it

**Comment by vhellstein** [comment]
@mpopke is going to look into how we might get close to the styling of our typical gallery text tombstone

**Comment by 1000camels** [comment]
Is the tombstone the name plate in the gallery?
I am happy to do that styling, but I was commenting that we have lost the title of the artwork in the md files, but perhaps I can get it from the objects.yaml.

**Comment by vhellstein** [comment]
Yes, sorry, that's correct. The titles of the artworks will be in the objects.yaml

**Comment by 1000camels** [comment]
No dimensions on your tomestone?

**Comment by 1000camels** [comment]
![Screenshot 2024-02-28 at 12 14 51 PM](https://github.com/DenverArtMuseum/biophilia-pub/assets/689727/7a2fac28-bd8a-43a9-890f-96083a61334a)
I've made a pass at this, but I am confused as you have said no artist or medium. Not sure what it should look like without an artist.
The title taken from the md file is a combined artist - title, which I can hide.

**Comment by vhellstein** [comment]
> No dimensions on your tomestone?

In the galleries, we don't put dimensions (the thinking is that people can see how big something is in front of them), but we'll have them in the catalog.

**Comment by vhellstein** [comment]
A few more things on these:

- [ ] @1000camels can image and text halves be the same size?


![Image](https://github.com/DenverArtMuseum/biophilia-pub/assets/119438504/b1dcf8f2-1320-4b68-9435-cb6211002007)

- [ ] @vhellstein to add link to online collection when applicable
- [ ] @vhellstein to double check all images against image log
- [ ] @vhellstein to fix paragraph split and image placement in Laarman/Nervous System entry.
- [ ] @mpopke need to address two image situation in Laarman/Nervous System entry

**Comment by 1000camels** [comment]
I am not seeing the new zoomable images for some reason. Can anyone confirm that they are seeing them?

**Comment by mpopke** [comment]
A problem we've noticed with the new Quire: When using zoomable images, the preview doesn't refresh properly. Images disappear or become empty black boxes when you save a change while preview is active and the page refreshes. In order to see the images you have to stop the preview server and start it again. I don't know if that's what you are seeing, or if it's something else.

**Comment by 1000camels** [comment]
I found documentation that said I should delete _site and public and rerun
quire preview. That fixed it for me.
